### ISSUE TITLE
Providing notification setting for message app

### ISSUE THREAD

**Issue by gzqx** [ubports/ubuntu-touch#2079]
### Description of the feature

Notification setting for message app liked discussed at the end of #1026

**Comment by lduboeuf** [comment]
I think a first approach would be to have a global toggle to disable notifications when screen locked. In Notification sestting

**Comment by gzqx** [comment]
This option is currently available, but hiding at System Settings - Security & Privacy - Locking and unlocking - when locked, allow notifications.

Putting it to a more intuitive place would indeed be a good start.

**Comment by lduboeuf** [comment]
ok i see. I would have place it in notifications , but idk if it easy to do there.

**Comment by lduboeuf** [comment]
oh ok i've not read the previous ticket correctly, so you would like to have a toggle specific to messaging app ? 
But that is not as easy than it seems.

**Comment by gzqx** [comment]
As I understand it, the origin post wants a specific lock screen toggle. 

However as me and the comment I replied to, we realized that messaging app is an abnormal, that unlike any other app in ubports, it does not have a notification setting. Thus, we feel like maybe we should bring at least a normal setting panel first.

**Comment by gzqx** [comment]
![image](https://user-images.githubusercontent.com/24262751/204887047-e0df2e3d-afaf-4bc3-8b19-3e9946971549.png)


As attached screenshot, everything but message shows here, which is weird.

**Comment by lduboeuf** [comment]
yes, but messaging app is a bit more than a normal app and uses notification not the same way.
Will try to look at it if i can
